crates/ruff_linter/resources/test/fixtures/flake8_comprehensions/C417.py

```diff
@@ -6,20 +6,20 @@
 set(map(lambda x: x % 2 == 0, nums))
 dict(map(lambda v: (v, v**2), nums))
 dict(map(lambda v: [v, v**2], nums))
-map(lambda: "const", nums)
+
 map(lambda _: 3.0, nums)
 _ = "".join(map(lambda x: x in nums and "1" or "0", range(123)))
 all(map(lambda v: isinstance(v, dict), nums))
 filter(func, map(lambda v: v, nums))
-list(map(lambda x, y: x * y, nums))
+
 
 # When inside f-string, then the fix should be surrounded by whitespace
 _ = f"{set(map(lambda x: x % 2 == 0, nums))}"
 _ = f"{dict(map(lambda v: (v, v**2), nums))}"
 
 # False negatives.
 map(lambda x=2, y=1: x + y, nums, nums)
-set(map(lambda x, y: x, nums, nums))
```

Comment:
this one actually doesn't raise an exception at runtime (possibly because multiple iterables are being passed in after the function?). It would be great if we could keep emitting a diagnostic for this one.

Reply:
I see. In that case, I probably wouldn't have moved this line in the fixture, since it's already under a line that says `# false negatives`. But it's not the biggest issue in the world.

Reply:
Actually, could you possibly move it back? It doesn't belong under the `# https://github.com/astral-sh/ruff/issues/15796` comment you've moved it under, because that issue is specifically about `map()` calls that would raise exceptions which Ruff erroneously replaces with code that doesn't. But that doesn't apply to this line.